server.test.js

```diff
@@ -131,6 +131,47 @@ describe('server', function() {
     });
   });
 
+  describe('makeGuess', function() {
+
+    it('should be able to guess letters', async function(done) {
+      const res = await request(server)
+                        .post('/')
+                        .send({
+                          "act": "guess",
+                          "guess": "d",
+                          "id": JSON.stringify(guesserID)
+                        });
+      expect(res.statusCode).toEqual(200);
+      expect(res.body.display).toEqual(['d','_','_','_','_']);
+      expect(res.body.guesses).toEqual(6);
+      expect(res.body.isOver).toEqual(false);
+      expect(game.wordToGuess).toEqual('debug');
+      done();
```

Comment:
Were we also going to add something like `attempts` into the response?

Reply:
We sure could! Let me know if that's something that feels good to do.